packaging/bundle-validation/base/build_flink1190hive313spark350.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
```

Comment:
cc @yihua , can you help with building these docker images?

Reply:
Yes.  These bundle-validation docker images are built and uploaded:
- [flink1190hive313spark351scala213](https://hub.docker.com/layers/apachehudi/hudi-ci-bundle-validation-base/flink1190hive313spark351scala213/images/sha256-3221553bec43f4e1e1afff98b2e9aebc61c12c8546ecaf6cac5f7e9d0480a9a4?context=explore)
- [flink1190hive313spark350](https://hub.docker.com/layers/apachehudi/hudi-ci-bundle-validation-base/flink1190hive313spark350/images/sha256-531e4cd3d91cd81095d39b07f3856e816f26857afb2bcd0f0d0684ce6c878d1c?context=explore)
- [flink1190hive313spark350scala213](https://hub.docker.com/layers/apachehudi/hudi-ci-bundle-validation-base/flink1190hive313spark350scala213/images/sha256-f527cf59014abff1985bec497ec7771c26164b243a10576551d37aeb36a4c44f?context=explore)

I restarted the CI tasks to see if they can succeed.

Reply:
All docker-based tests pass now.